src/app/components/viewer/viewer.component.html

```diff
@@ -3,7 +3,7 @@
     [ngClass]="{
       'right_side--hide': !showRightSide
     }"
-    [displayName]="fileName"
+    [fileName]="fileName"
```

Comment:
does this mean the ADF has a change in the property?

Reply:
No denys, ADF support both displayName and fileName . 

displayName - is kind of static one. you can't update the name of the file once viwer opened.
fileName - update the view file name whenever node properites updated or new node passed

Reply:
Ok, thanks